### PR TITLE
[Concept List-Methods] Further edited language in about.md and introduction.md.

### PR DESCRIPTION
**Pull request by BethanyG**
Removed references to "you" and "you will" in docs and made the "voice"  less personal.

### CODE REVIEW

**Review by neenjaw — APPROVED** [review]
These wording changes look good!!

**Comment by neenjaw** [comment]
My approval means nothing in this repo officially, but thought would be worthwhile to review the wording changes.

File: concepts/list-methods/about.md [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
+They're considered a [sequence][sequence type] in Python, and can be copied in whole or in part via [slice notation][slice notation].
+Like any sequence, elements within `lists` can be referenced by `0-based index` number from the left, or `-1-based index` number from the right.
 
-Lists support both [common][common sequence operations] and [mutable][mutable sequence operations] sequence opterations like `min(<list>)`/`max(<list>)`, `<list>.index()`, `<list>.append()` and `<list>.reverse()`. Items can be iterated over using the `for item in <list>` construct, and `for item in enumerate(<list>)` can be used when both the element and index are needed.
+Lists support both [common][common sequence operations] and [mutable][mutable sequence operations] sequence opterations like `min(<list>)`/`max(<list>)`, `<list>.index()`, `<list>.append()` and `<list>.reverse()`.
+Items can be iterated over using the `for item in <list>` construct, and `for item in enumerate(<list>)` can be used when both the element and element index are needed.
```
Comment (BethanyG) [inline concepts/list-methods/about.md:8]:
```suggestion
Items can be iterated over using the `for item in <list>` construct, and `for index, item in enumerate(<list>)` can be used when both the element and element index are needed.
```

**Comment by neenjaw** [comment]
I am on the @exercism/reviewers team, just don't have approve powers here :)

**Pull request merged**